app/views/layouts/application.html.haml

```diff
@@ -5,15 +5,21 @@
     %title SlowFood
     = csrf_meta_tags
     = stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' =>'reload'
+    = javascript_include_tag 'http://maps.google.com/maps/api/js?key=AIzaSyDksFPPj-FomswnVnuVeNMdGBYb1nA4pas'
```

Comment:
We could run into problems with loading assets over http on Heroku ping @diraulo